### DISCUSSION TITLE
How can I randomize prompts? like "{1|2|3}" in 1111

### DISCUSSION THREAD

**Discussion by gamer2key** [lllyasviel/stable-diffusion-webui-forge#551] [Q&A] [answered]
As the title says, I would really like to use this feature on Forge but I don't know how

**Comment by catboxanon** [comment, marked as answer]
That syntax comes from an extension. https://github.com/adieyal/sd-dynamic-prompts

**Reply by gamer2key** [reply]
I was using that extension on 1111 without knowing, I'm stupid, thank you!
